### DISCUSSION TITLE
Anyone have issues with HDHR power supplies?

### DISCUSSION THREAD

**Discussion by wayner9** [OpenSageTV/Community#106] [General]
I had an HDHR that wasn't working - I couldn't see it on my LAN and SageTV couldn't connect to the tuners, although the power LED  was working.  I replaced the power supply  which is 5V and 2A, and it now seems good.  This is a very old unit that is gray coloured and has two antenna inputs.  It looks like this:  
![images](https://github.com/OpenSageTV/Community/assets/10366145/9460b648-212b-4db6-a5b0-67ae3aaba409)

Has anyone else had issues with HDHR power supplies in the past?

The Hauppauge HD-PVR was legendary for crappy power supplies, but I don't remember this issue being as prevalent with the HDHR.

**Comment by JustFred51** [comment]
I've been using a pair of those dual-channel HDHR tuners for over a decade.  The OEM supplies were good quality (unlike those from Hauppauge).  In the last couple of years, both power supplies have gotten flaky.  I replaced them with new (generic) power supplies (same voltage and current ratings).  The tuners continue to work like a champ.

**Comment by LoisGNS** [comment]
My HDHR (I think the same model as yours) had the same symptoms and was fixed in the same way. It still isn't nearly as good at tuning as my Hauppuage devices, but it's enabled for stronger OTA channels as tuner-of-least-merit.

**Reply by wayner9** [reply]
I also have a newer HDHR that has a single antenna input and it seems better at tuning weaker channels.  Somewhere I have a Hauppauge 950 stick tuner but I can't seem to find it, but I don't remember that as having been better than the newer HDHR.  And I used to have a Hauppauge PCI card tuner as well, but I never bothered putting that in my unRAID server, I wasn't even sure that it would work in Linux or whether it was totally obsolete.
